e2e/testcases/csr_auth_test.go

```diff
@@ -178,8 +185,8 @@ func getMembership(fleetMembership, gcpProject string) (bool, error) {
 		// - ERROR: (gcloud.container.fleet.memberships.describe) No memberships available in the fleet.
 		// - ERROR: (gcloud.container.fleet.memberships.describe) NOT_FOUND: Resource 'projects/gcpProject/locations/global/memberships/fleetMembership' was not found
 		if strings.Contains(out, fmt.Sprintf("Membership %s not found in the fleet", fleetMembership)) ||
-			strings.Contains(out, "No memberships available in the fleet") ||
-			strings.Contains(out, "NOT_FOUND") {
+				strings.Contains(out, "No memberships available in the fleet") ||
```

Comment:
why is this indented? wont it fail formating?

Reply:
IDE autoformatted this, I'll revert

Reply:
done